src/hooks/verifyInstallSignature.ts

```diff
@@ -26,15 +26,17 @@ export class VerificationConfigBuilder {
     const vConfig = new VerificationConfig();
     vConfig.verifier = new InstallationVerification().setPluginNpmName(npmName).setConfig(configContext);
 
-    vConfig.log = cli.log.bind(cli) as (msg: string) => void;
+    vConfig.log = (msg: string): void => {
+      CliUx.ux.log(msg);
+    };
```

Comment:
wraps cliux in an arrow fn instead of messing with `this`.

Reply:
you can't touch this.